tap_gitlab/__init__.py

```diff
@@ -185,6 +195,29 @@ def sync_users(project):
             singer.write_record("users", transformed_row, time_extracted=utils.now())
 
 
+def sync_group(gid, pids):
+    url = CONFIG['api_url'] + RESOURCES["groups"]['url'].format(gid)
+
+    data = request(url).json()
+    time_extracted = utils.now()
+
+    with Transformer(pre_hook=format_timestamp) as transformer:
```

Comment:
Its a good practice to limit the scope of the Transformer for as long as its needed. In this case, I'd like to see the code after the transformation be unindented so the Transformer is used and discarded as the tap moves forward into the other for loop.

Reply:
I did so, and also applied it to sync_project. Please check if this what you expected.